scripts/breaking_changes_checker/breaking_changes_tracker.py

```diff
@@ -92,6 +92,7 @@ def __init__(self, stable: Dict, current: Dict, package_name: str, **kwargs: Any
         self.stable = stable
         self.current = current
         self.diff = jsondiff.diff(stable, current)
+        self.features_added = []
```

Comment:
Technically I think we will eventually move to having 1 changes list for all of the checkers specified on an instance of the tracker class but we can think more about what this will look like once the BreakingChangesTracker class is refactored to a more generic CodeReporter class.